packages/dx-react-scheduler-material-ui/src/templates/common/container.jsx

```diff
@@ -2,16 +2,21 @@ import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import classNames from 'clsx';
 
+const PREFIX = 'Container';
+
+export const classes = {
+  container: `${PREFIX}-container`,
+};
+
 export const ContainerBase = ({
-  children, classes, className, ...restProps
+  children, className, ...restProps
 }) => (
   <div className={classNames(classes.container, className)} {...restProps}>
```

Comment:
There should be styled div

Reply:
Moved styles here from packages\dx-react-scheduler-material-ui\ src\ templates\ layout.jsx